.gitpod.Dockerfile

```diff
@@ -11,6 +12,7 @@ RUN apt-get install -yqq wget
 RUN apt-get install -yqq ca-certificates
 RUN apt-get install -yqq ssh
 RUN apt-get install -yqq git
+RUN apt-get install -yqq sudo
 
 RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
what does apt-get clean and rm do?

Reply:
`apt-get clean` removes left over files that aren't needed anymore after installing packages via apt.
rm deteles a file, in this case `rm -rf /var/lib/apt/lists` is removing the list of packages generated from `apt update`.

I believe the purpose of this line is to free up disk space.